features/items.feature

```diff
@@ -109,4 +109,23 @@ Scenario: Delete an item
     And the item "Price" field should be empty
     When I paste the item "Shopcart Id" field
     And I press the item "Retrieve" button
-    Then I should see the message "is empty"
\ No newline at end of file
+    Then I should see the message "is empty"
+
+Scenario: Update an item
```

Comment:
You have some of the wrong button names and forgot to use the item steps instead of the shopcart steps in a few places. Try something like:

Scenario: Update an item
    When I visit the "Home Page"
    And I set the item "Name" to "item_2"
    And I press the item "Query by Name" button
    Then I should see "item_2" in the item "Name" field
    And I should see "59.99" in the item "Price" field
    When I set the item "Name" to "item_4"
    And I press the item "Update" button
    Then I should see the message "Success"
    When I copy the item "Product Id" field
    And I press the item "Clear" button
    And I paste the item "Product Id" field
    And I press the item "Query by Product Id" button
    Then I should see "item_4" in the item "Name" field
    When I copy the item "Shopcart Id" field
    And I press the "Clear" button
    And I paste the item "Id" field
    And I press the item "Retrieve" button
    Then I should see "item_4" in the item results
    Then I should not see "item_2" in the item results